utils/package.json

```diff
@@ -50,7 +50,7 @@
         "@types/mocha": "^7.0.2",
         "@types/node": "^16.0.0",
         "@types/semver": "^7.3.9",
-        "@types/vscode": "1.64.0",
+        "@types/vscode": "^1.74.0",
```

Comment:
Should we just do 1.76 instead, so that we don't run into problems, if it was introduced in 1.74 but not available until 1.76?